src/Lykke.Job.SiriusCashoutProcessor/Services/CashoutProcessorService.cs

```diff
@@ -173,8 +174,8 @@ private async Task ProcessCashoutsAsync()
                                 new
                                 {
                                     siriusWithdrawalId = item.Withdrawal.Id,
-                                    clientId = item.Withdrawal.UserNativeId,
-                                    walletId = item.Withdrawal.AccountReferenceId == item.Withdrawal.UserNativeId ? item.Withdrawal.UserNativeId : item.Withdrawal.AccountReferenceId,
+                                    clientId = item.Withdrawal.GetUserNativeId(),
+                                    walletId = item.Withdrawal.AccountReferenceId == item.Withdrawal.GetUserNativeId() ? item.Withdrawal.GetUserNativeId() : item.Withdrawal.AccountReferenceId,
```

Comment:
instead of `item.Withdrawal.AccountReferenceId` it should be wallet id from the properties

Reply:
and check it everywhere

Reply:
it would be good to implement it as an extension method with fallback to legacy approach as you did for the user native ID